src/index.html

```diff
@@ -5,8 +5,380 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="nav">
+      <a href="#" class="nav__logo">
+        <img src="images/logo.png" alt="Logo">
```

Comment:
Logo is not Nav element its header element, you need to have header__logo

Reply:
![image](https://user-images.githubusercontent.com/105974422/180203720-d788e14e-dfeb-40d9-8668-e482ba62d051.png)